src/alacrity.rs

```diff
@@ -13,7 +15,19 @@ fn main() {
     let addr = addr.parse::<SocketAddr>().unwrap();
 
     let backend = env::args().nth(2).unwrap_or("127.0.0.1:12345".to_string());
-    let pool = Pool::new(vec![backend]).unwrap();
+    let backend = backend.parse::<SocketAddr>().unwrap();
+    let pool = Pool::with_servers(vec![backend]);
 
-    alacrity::proxy::listen(addr, pool.clone()).expect("Failed to start server")
+    let admin_ip = env::args().nth(2).unwrap_or("127.0.0.1:8687".to_string());
+    let admin_addr = admin_ip.parse::<SocketAddr>().unwrap();
+    let p = pool.clone();
+    let _ = thread::Builder::new().name("management".to_string()).spawn(move || {
+        mgmt::listen(admin_addr, p);
+    }).expect("Failed to create proxy thread");
+
+    let h = thread::Builder::new().name("proxy".to_string()).spawn(move || {
+        alacrity::proxy::listen(addr, pool.clone()).expect("Failed to start server")
+    }).expect("Failed to create proxy thread");
+
+    let _ = h.join();
```

Comment:
For curiosity: why should the proxy runs in its own thread?
Should we "waste" one thread for that?
Should the proxy use a thread pool in the future (with # threads = # cores)? (or one proxy per thread that can share a listening port?)